Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -293,6 +293,19 @@ Examples of Commands:
    $cmd['tt_content'][1203]['move'] = 400;  // Moves tt_content uid=1203 to the first position in page uid=400
 
 
+.. _tce-data:
+
+Accessing the uid of copied records:
+---------------------
+
+::
+   $cmd['tt_content'][1203]['copy'] = 400;  // Copies tt_content uid=1203 to first position in page uid=400
+   $dataHandler->start([], $cmd);
+   $dataHandler->process-cmdmap()
+   
+   $uid = $dataHandler->copyMappingArray_merged['tt_content]['1203];
```

Comment:
```suggestion
   $uid = $dataHandler->copyMappingArray_merged['tt_content][1203];
```

Small syntax error.

Maybe you can also mention `copyMappingArray` and the difference between these two.